Revert "Notify user when connecting as follower fails (#19477)" (#20131)
nvaccess/nvda: user_docs/en/changes.md

Reviewer: ```suggestion
This is a patch release to fix a security issue.
We have also removed a change we introduced in 2026.1 with how we handle connection issues with Remote Access.
```

## 2026.1.1

This is a patch release to fix a security issue.

> We have also removed a change we introduced in 2026.1 with how we handle connection issues with Remote Access.

Reviewer: ```suggestion
A change introduced in 2026.1 with how NVDA handles connection issues with Remote Access was removed.
```
